src/client/Microsoft.Identity.Client/OAuth2/MsalTokenResponse.cs

```diff
@@ -191,35 +191,36 @@ public void Log(ICoreLogger logger, LogLevel logLevel)
                 StringBuilder withPii = new StringBuilder();
                 StringBuilder withoutPii = new StringBuilder();
 
-                withPii.AppendLine("==MsalTokenResponse==");
-                withoutPii.AppendLine("==MsalTokenResponse==");
-
-                withPii.AppendLine($"Error: {Error} ErrorDescription: {ErrorDescription}");
-                withoutPii.AppendLine($"Error: {Error} ErrorDescription: {ErrorDescription}");
+                withPii.AppendLine($"{Environment.NewLine}[MsalTokenResponse]");
+                withPii.AppendLine($"Error: {Error}");
+                withPii.AppendLine($"ErrorDescription: {ErrorDescription}");
                 withPii.AppendLine($"Scopes: {Scope} ");
+                withPii.AppendLine($"ExpiresIn: {ExpiresIn}");
+                withPii.AppendLine($"RefreshIn: {RefreshIn}");
+                withPii.AppendLine($"AccessToken returned: {!string.IsNullOrEmpty(AccessToken)}");
+                withPii.AppendLine($"AccessToken Type: {TokenType}");
```

Comment:
TokenType is not Pii

Reply:
sorry, it's hard to review w/the lines changed and matching up what's pii and what's not. i thought our logs didn't have to be repeated anymore, and the pii logs would include the non-pii.

Reply:
Ignore this comment. Not applicable to the fix anyway.